src/doc/guide.md

```diff
@@ -83,18 +83,17 @@ $ ./target/debug/hello_world
 Hello, world!
 ```
 
-We can also use `cargo run` to compile and then run it, all in one step:
+We can also use `cargo run` to compile and then run it, all in one step (You
+won't see the `Compiling` line if you have not made any changes since you last
+compiled):
 
 <pre><code class="language-shell"><span class="gp">$</span> cargo run
 <span style="font-weight: bold"
-class="s1">     Fresh</span> hello_world v0.1.0 (file:///path/to/project/hello_world)
+class="s1">   Compiling</span> hello_world v0.1.0 (file:///path/to/project/hello_world)
 <span style="font-weight: bold"
 class="s1">   Running</span> `target/debug/hello_world`
 Hello, world!</code></pre>
 
-To pass some arguments to your program, use `cargo run first_arg second_arg`.  
-If flags are being passed, use a “--” separator to tell Cargo which flags go where, like `cargo run -- --foo -b bar`.
```

Comment:
Perhaps it can be moved to the example with the `regex` crate? Hardcoded `2014-01-01` looks a bit old school, instead the date can be read from the command line.

On the other hand, I am not sure that this is that valuable info...